pkg/services/ngalert/api/tooling/definitions/cortex-ruler.go

```diff
@@ -402,22 +405,23 @@ type PostableGrafanaRule struct {
 
 // swagger:model
 type GettableGrafanaRule struct {
-	ID              int64               `json:"id" yaml:"id"`
-	OrgID           int64               `json:"orgId" yaml:"orgId"`
-	Title           string              `json:"title" yaml:"title"`
-	Condition       string              `json:"condition" yaml:"condition"`
-	Data            []AlertQuery        `json:"data" yaml:"data"`
-	Updated         time.Time           `json:"updated" yaml:"updated"`
-	IntervalSeconds int64               `json:"intervalSeconds" yaml:"intervalSeconds"`
-	Version         int64               `json:"version" yaml:"version"`
-	UID             string              `json:"uid" yaml:"uid"`
-	NamespaceUID    string              `json:"namespace_uid" yaml:"namespace_uid"`
-	NamespaceID     int64               `json:"namespace_id" yaml:"namespace_id"`
-	RuleGroup       string              `json:"rule_group" yaml:"rule_group"`
-	NoDataState     NoDataState         `json:"no_data_state" yaml:"no_data_state"`
-	ExecErrState    ExecutionErrorState `json:"exec_err_state" yaml:"exec_err_state"`
-	Provenance      Provenance          `json:"provenance,omitempty" yaml:"provenance,omitempty"`
-	IsPaused        bool                `json:"is_paused" yaml:"is_paused"`
+	ID              int64        `json:"id" yaml:"id"`
+	OrgID           int64        `json:"orgId" yaml:"orgId"`
+	Title           string       `json:"title" yaml:"title"`
+	Condition       string       `json:"condition" yaml:"condition"`
+	Data            []AlertQuery `json:"data" yaml:"data"`
+	Updated         time.Time    `json:"updated" yaml:"updated"`
+	IntervalSeconds int64        `json:"intervalSeconds" yaml:"intervalSeconds"`
+	Version         int64        `json:"version" yaml:"version"`
+	UID             string       `json:"uid" yaml:"uid"`
+	NamespaceUID    string       `json:"namespace_uid" yaml:"namespace_uid"`
+	// Deprecated: it will be removed in a future release in favor of NamespaceUID
+	NamespaceID  int64               `json:"namespace_id" yaml:"namespace_id"`
```

Comment:
@grafana/alerting-frontend I wonder if it's possible to get rid of it now.

Reply:
We're not using it, we should be ok to remove it.

Reply:
This has been removed in a separate PR. I've rebased this PR on top, we no longer need to worry about this here.